plugins/wasi_nn/CMakeLists.txt

```diff
@@ -1,102 +1,120 @@
 # SPDX-License-Identifier: Apache-2.0
 # SPDX-FileCopyrightText: 2019-2022 Second State INC
 
-# llama.cpp options
-# Disable warnings and debug messages
-set(LLAMA_ALL_WARNINGS OFF)
-set(LLAMA_METAL_NDEBUG ON)
-
-if(WASMEDGE_PLUGIN_WASI_NN_GGML_LLAMA_CUBLAS)
-  message(STATUS "WASI-NN GGML LLAMA backend: Enable LLAMA_CUBLAS")
-  set(LLAMA_CUBLAS ON)
-  # If CUBLAS is ON, then OpenBLAS should be OFF.
-  set(WASMEDGE_PLUGIN_WASI_NN_GGML_LLAMA_BLAS OFF)
-else()
-  message(STATUS "WASI-NN GGML LLAMA backend: Disable LLAMA_CUBLAS")
-  set(LLAMA_CUBLAS OFF)
-endif()
+string(TOLOWER ${WASMEDGE_PLUGIN_WASI_NN_BACKEND} BACKEND)
+if(BACKEND STREQUAL "ggml")
+  # llama.cpp options
+  # Disable warnings and debug messages
+  set(LLAMA_ALL_WARNINGS OFF)
+  set(LLAMA_METAL_NDEBUG ON)
+  set(LLAMA_STATIC ON)
```

Comment:
Will this option make cublas static link? If so, we should disable it.

Reply:
This option causes our plugin to statically link libcublas. I will disable it.